packages/parser/lib/Tokens.re

```diff
@@ -99,13 +99,18 @@ let token_to_string =
   | INTERPOLATION(v) => String.concat(".", v)
   | WS => " "
   | DOT => "."
-  | COMMA => ","
+  | COMMA => "COMMA"
   | ASTERISK => "*"
   | FUNCTION(fn) => fn ++ "("
   | NTH_FUNCTION(fn) => fn ++ "("
   | URL(url) => url ++ "("
   | BAD_URL => "bad url"
-  | BAD_IDENT => "bad indent";
+  | BAD_IDENT => "bad indent"
+  | PERCENTAGE(p) =>
+    "percentage: " ++ string_of_float(p) ++ string_of_char('%')
+  | NUMBER_CSS(n) => "(NUMBER '" ++ string_of_float(n) ++ "')"
+  | DIMENSION_CSS((f, s)) => "dimension: " ++ string_of_float(f) ++ s
```

Comment:
Does it make sense for `NUMBER_CSS` and `DIMENSION_CSS` to be produced as strings as `NUMBER` and `DIMENSION` are stored as strings? In that case, we could remove `NUMBER_CSS` and `DIMENSION_CSS`.  tokens

Reply:
I don't see the difference (either I understand) NUMBER vs NUMBER_CSS.

Why do we need both? Why we need another?

Reply:
We could have one if NUMBER_CSS and DIMENSION_CSS are produced as strings